torch/cuda/__init__.py

```diff
@@ -215,6 +215,11 @@ def current_stream():
     return torch.cuda.Stream(_cdata=torch._C._cuda_getCurrentStream())
 
 
+def current_blas_handle():
+    """Returns cublasHandle_t pointer to current cuBLAS handle"""
```

Comment:
You need to register that function in `docs/source`.

Also, a nit - add a dot at the end of the sentence.